recipes/injective-py/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+version:
+  - 0.9.12
```

Comment:
Please build different versions as separate branches on the feedstock and not via this way.

Reply:
Sure.